operating_unit/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2015-TODAY Eficent
+# - Jordi Ballester Alomar
+# Copyright 2015-TODAY Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License: LGPL-3 or later (https://www.gnu.org/licenses/lgpl.html).
+{
+    "name": "Operating Unit",
+    "summary": "An operating unit (OU) is an organizational entity part of a "
+    "company",
+    "version": "13.0.1.0.0",
+    "author": "Eficent, "
+    "Serpent Consulting Services Pvt. Ltd.,"
+    "Jarsa Sistemas, "
```

Comment:
For migrating modules without adding any major feature putting yourself in the contributors section in the readme should be enough. This is reserved for the original author + contributors who do major functional changes.

Hope you understand.

Reply:
Thanks for the clarification 👍
I already make the change

Reply:
Thank you!